site/docs/guides/flowctl/create-derivation.md

```diff
@@ -31,17 +31,13 @@ If you need help, see the [guide to create a Data Flow](./create-dataflow.md).
 To begin working in your local environment, you must authenticate Flow from the command line.
 Then, you'll need to add your source collection's specification files to a **draft** and bring the draft into your local environment for editing.
 
-1. In your local environment, authenticate flowctl:
+1. Authorize flowctl.
 
-   ```console
-   flowctl auth login
-   ```
-
-   A browser window opens to the CLI-API tab of the Flow web app.
+   1. Go to the [CLI-API tab of the web app](https://dashboard.estuary.dev/admin/api) and copy your access token.
 
-2. Copy the access token and paste it in the Auth Token prompt in your terminal. Press Enter.
+   2. Run `flowctl auth token --token <paste-token-here>`
 
-3. Begin by creating a fresh draft. This is where you'll add the specification files you need from the catalog.
+2. Begin by creating a fresh draft. This is where you'll add the specification files you need from the catalog.
```

Comment:
I think these steps should change for the latest flowctl. There's no longer a need to use a draft in order to pull specs, and there's also no longer a need to pull the source collection specs in the first place. Pretty much all of the steps between here and step 3 of `Add a derivation` are no longer necessary. And I think the stuff about finding the specific file to put your derivation into should perhaps be replaced with a slightly more generic instruction of how to name the file based on the desired name of your derivation, since there likely won't be existing specs in their directory. Totally fine to do this in a subsequent PR, though.

Reply:
Noted - I wasn't looking at this page too hard here but I should do so soon, and will make these changes at that point.